addon/components/base/bs-collapse.js

```diff
@@ -13,12 +13,12 @@ import { assert } from '@ember/debug';
  ### Usage
 
  ```hbs
- {{#bs-collapse collapsed=collapsed}}
-   <div class="well">
-     <h2>Collapse</h2>
-     <p>This is collapsible content</p>
-   </div>
- {{/bs-collapse}}
+  <BsCollapse @collapsed={{this.collapsed}} >
```

Comment:
Ah, yes, you also refactor this to use `this.`, this is nice!

Reply:
Yes, octane style templates is just a peace of mind... actually nowadays I can't stand old templates, like _where_is_this_value_coming_from? 👍